pkg/api/openapi/minder/v1/minder.swagger.json

```diff
@@ -1418,6 +1418,50 @@
         "tags": [
           "ProvidersService"
         ]
+      },
+      "patch": {
+        "operationId": "ProvidersService_PatchProvider",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/v1PatchProviderResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "patch",
```

Comment:
I'm assuming this name doesn't show up anywhere.

Reply:
It does not, e.g. one calls the endpoint as:
```
 curl -XPATCH -H "Authorization: Bearer $MINDER_BEARER_TOKEN" 'http://localhost:8080/api/v1/providers?context.project=bcf5bc68-aa81-4f59-96ae-7dceb5c3663f&context.provider=github-app-jakubtestorg' -d '{ "config": { "auto_registration": { "enabled": ["repository"] }} }'
```
in other words, the body already contains the provider attributes.